services/device-connection/src/main/java/org/eclipse/hono/deviceconnection/infinispan/Application.java

```diff
@@ -36,10 +36,9 @@
  * The application implements Hono's <a href="https://www.eclipse.org/hono/docs/api/device-connection/">
  * Device Connection API</a>.
  */
-@ComponentScan("org.eclipse.hono.deviceconnection.infinispan")
 @ComponentScan("org.eclipse.hono.service.auth")
 @ComponentScan("org.eclipse.hono.service.metric")
-@Configuration
+@Import(value = { ApplicationConfig.class, EmbeddedCacheConfig.class, RemoteCacheConfig.class })
```

Comment:
You can leave out the `value = ` part, as this is the default name and can be omitted.